fetch.bs

```diff
@@ -5232,6 +5348,10 @@ these steps:
   <a>CORS check</a>, as <var>request</var>'s <a for=request>client</a> and the service worker can
   have different embedder policies.
 
+ <li>If <var>request</var>'s <code>https-upgrade</code> flag is set, set <var>response</var> and
+  <var>internalResponse</var> to the result of running
+  <a>HTTPS upgrade fallback</a> given <var>request</var> and <var>response</var>.
```

Comment:
Indentation is wrong.

Reply:
Done

Reply:
It doesn't look like this is done?

Reply:
Was done locally, forgot to push. Done now?